src/test/java/redis/clients/jedis/tests/commands/PublishSubscribeCommandsTest.java

```diff
@@ -65,6 +65,27 @@ public void onUnsubscribe(String channel, int subscribedChannels) {
 
   @Test
   public void pubSubChannels() {
+    final List<String> expectedActiveChannels = Arrays.asList("testchan1", "testchan2",
+      "testchan3", "__sentinel__:hello");
+    jedis.subscribe(new JedisPubSub() {
+      private int count = 0;
+
+      @Override
+      public void onSubscribe(String channel, int subscribedChannels) {
+        count++;
+        // All channels are subscribed
+        if (count == 3) {
+          Jedis otherJedis = createJedis();
+          List<String> activeChannels = otherJedis.pubsubChannels();
+          assertTrue(expectedActiveChannels.containsAll(activeChannels));
```

Comment:
Add the comment in the following test as well.

```suggestion
          List<String> activeChannels = otherJedis.pubsubChannels();
          // Can't use 'assertEquals'. Since we are utilizing sentinel for the tests, there is an additional '__sentinel__:hello' channel that has subscribers.
          assertTrue(expectedActiveChannels.containsAll(activeChannels));
```

Reply:
I think we'd have to use `containsAll` either way since the list of channels may come back in a different order than the expected list.  We also use `containsAll` in the `pubSubChannelsWithPattern` test below this one.  I added the comment, but put it right above the assignment of the `expectedActiveChannels` variable.  

Alternatively, we could use `contains` instead of `containsAll` and remove the sentinel channel from the expected list.

Let me know what you think!

Reply:
Between these two, I prefer it as it is.

Reply:
Sounds good, I will leave it as it is.